modules/reference/pages/console/record-deserialization.adoc

```diff
@@ -1,27 +1,171 @@
-= Record Deserialization
-:description: Learn how record deserialization in Redpanda Console works and what encodings it supports.
-:page-aliases: console:features/record-deserialization.adoc
+= Deserialization in Redpanda Console
+:description: Learn how Redpanda Console deserializes messages.
+:page-aliases: console:features/record-deserialization.adoc, manage:console/protobuf.adoc
 
-When consuming messages, Redpanda Console's message viewer is capable of automatically deserializing messages.
-It identifies the correct deserialization type by trying to decode the record's key, value, or header with all
-available deserialization methods. The supported deserialization methods are:
+In Redpanda, the messages exchanged between producers and consumers contain raw bytes. Schemas work as an agreed-upon format, like a contract, for producers and consumers to serialize and deserialize those messages. If a producer breaks this contract, consumers can fail. 
 
-* Kafka's internal binary formats (for example the `__consumer_offsets` topic )
+Redpanda Console automatically tries to deserialize incoming messages and show them in human-readable format. It tests different deserialization strategies until it finds one with no errors. If no deserialization attempts are successful, Redpanda Console renders the byte array in a hex viewer. Sometimes it may be expected to see the payload in hex bytes (for example, because the payload is encrypted or because it uses a serializer that Redpanda Console cannot deserialize). When this happens, Redpanda Console displays troubleshooting information. This can be a helpful tool to debug applications that consume messages from that topic. You can also download the raw bytes of the message to feed it directly to your client deserializer or share it with support. 
```

Comment:
Sorry for being picky and possibly confusing before with my other comments, but I would just remove this sentence I believe because I still think it may be confusing to some:

> This can be a helpful tool to debug applications that consume messages from that topic.